ipaserver/plugins/domainlevel.py

```diff
@@ -48,6 +48,30 @@ def get_domainlevel_range(master_entry):
         return DomainLevelRange(0, 0)
 
 
+def check_conflict_entries(ldap, api):
+    """
+    Returns true if conflict entries exist in topology subtree
+    """
+
+    container_dn = DN(
+        ('cn', 'ipa'),
+        ('cn', 'etc'),
+        api.env.basedn
+    )
+
+    try:
+        conflicts, _dummy = ldap.find_entries(
+            filter="(&(nsds5replconflict=*)(|(objectclass=ldapsubentry)(objectclass=*)))",
+            base_dn=container_dn,
+            scope=ldap.SCOPE_SUBTREE)
+    except errors.EmptyResult:
+        conflicts = None
```

Comment:
I would rewrite the body of this function like this:

```
+    try:
+        ldap.get_entries(
+            filter="(&(nsds5replconflict=*)(|(objectclass=ldapsubentry)(objectclass=*)))",
+            base_dn=container_dn,
+            scope=ldap.SCOPE_SUBTREE)
+            return True
+    except errors.NotFound:
+        return False
```

`get_entries` raises NotFound if the result is empty. This makes the code little more pythonic IMHO.